src/reporter.ts

```diff
@@ -241,56 +241,47 @@ class RPReporter implements Reporter {
   }
 
   onTestBegin(test: TestResp): void {
-    //create suite
-    const suiteHasParent = test.parent.parent?._isDescribe;
-    const suiteTitle = suiteHasParent ? test.parent.parent?.title : test.parent.title;
-    if (!this.findTestItem(this.suites, suiteTitle)) {
-      const codeRef = getCodeRef(test, TEST_ITEM_TYPES.SUITE);
+    const suitesOrder: string[] = [];
+    const createSuitesOrder = (suite: any) => {
+      if (!suite?._isDescribe) {
+        return;
+      }
```

Comment:
Please check possibility do not using "private" property `_isDescribe` it can change at any time